src/open_inwoner/templates/pages/search.html

```diff
@@ -24,26 +25,31 @@ <h2 class="utrecht-heading-2">{% trans "Zoekfilters" %}</h2>
                 {% if search_filter_organizations %}
                     {% include "components/Filter/Filter.html" with field=search_form.organizations form_id="search-form" only %}
                 {% endif %}
-        {%  endif %}
+            </aside>
+        {% endif %}
         {#  end search filters #}
-        </aside>
+    {% endif %}
 
         <div class="grid__main grid__main--search">
             {% if paginator.count %}
                 <div class="search-results">
-                    <h2 class="utrecht-heading-2 search-results__title">
-                        {% trans "Zoekresultaten" %}
-                        <span class="search-results__title-small">{{paginator.count}} zoekresultaten</span>
-                    </h2>
                     <div class="search-results__list">
-                        {% for hit in page_obj %}
-                            <a class="search-results__item" href="{% url 'products:product_detail' hit.slug %}">
-                                <h3 class="utrecht-heading-3 search-results__item-title">{{ hit.name }}</h3>
-                                <div class="search-results__item-info-container">
-                                    <p class="search-results__item-intro">{{ hit.summary }}</p>
+                            {% render_grid %}
+                                {% for hit in page_obj %}
+                                <div class="column column--start-0 {% if search_filter_categories or search_filter_tags or search_filter_organizations %}column--span-12{% else %}column--span-8{% endif %}">
```

Comment:
```suggestion
                                <div class="column column--start-0 column--span-{{ 12 if search_filter_categories or search_filter_tags or search_filter_organizations else 8 }}">
```

This if else could be shorter and more readable.

Reply:
@stefrado That will not work, Django HTML templates do not support inline Python expressions like that; the curly braces `{{ }} `are meant only for variable output, not logic processing.
I could change it into this: `<div class="column column--start-0 column--span-{% if search_filter_categories or search_filter_tags or search_filter_organizations %}12{% else %}8{% endif %}">`
which is a minor win.

Reply:
Yess you are correct 🏆